src/helpers/calculations.test.js

```diff
@@ -0,0 +1,35 @@
+import { getLifeCount } from './calculations';
+
+const age = 25;
+
+test('life counts returns 90 yeart without maxAge parameter', () => {
+  expect(getLifeCount(new Date()).years).toBe(90);
+});
+
+test('life counts returns 1080 months without maxAge parameter', () => {
+  expect(getLifeCount(new Date()).months).toBe(1080);
+});
+
+test('life count returns 4696 weeks without maxAge parameter', () => {
+  expect(getLifeCount(new Date()).weeks).toBe(4696);
+});
+
+test("life count throwing TypeError if first parameter isn't date object", () => {
+  let error = null;
+  try {
+    getLifeCount({});
+  } catch (e) {
+    error = e;
+  }
+  expect(error instanceof TypeError).toBe(true);
```

Comment:
there is a better way )
https://jestjs.io/docs/en/expect.html#tothrowerror

Reply:
I tried, but here is what I've got
![1](https://user-images.githubusercontent.com/37845039/84912958-aa911400-b0c2-11ea-9e53-c11b1865c3b8.png)

Reply:
That means that your test is failing.
Could you debug what lived value do you have in this case? 
What does the function return?

Reply:
I see the problem now.
Dangerous code should be wrapped into a function, like this
```javascript
test.only("life count throwing Error if date is in the future", () => {
  expect(() => {
    getLifeCount(new Date(Date.now() + 1));
  }).toThrowError("Seems like you have a time machine");
});
```
Btw, if you are providing an exact message with the error, would be nice to test it as well